config/src/fuzz.rs

```diff
@@ -35,6 +39,34 @@ impl Default for FuzzConfig {
     }
 }
 
+impl InlineConfigParser for FuzzConfig {
+    fn config_key() -> String {
+        INLINE_CONFIG_FUZZ_KEY.into()
+    }
+
+    fn try_merge(&self, configs: &[String]) -> Result<Option<Self>, InlineConfigParserError> {
+        let overrides: Vec<(String, String)> = Self::get_config_overrides(configs);
+
+        if overrides.is_empty() {
+            return Ok(None)
+        }
+
+        // self is Copy. We clone it with dereference.
+        let mut conf_clone = *self;
```

Comment:
ok so if i understand correctly this is a copy that we're later mutating and returning? can we do `self.clone()`? This doesn't seem too idiomatic